pkg/hub/submarinerbrokerinfo/brokerinfo_test.go

```diff
@@ -167,7 +167,7 @@ var _ = Describe("Function Get", func() {
 		When("no SubmarinerConfig is provided", func() {
 			It("should return the defaults", func() {
 				Expect(brokerInfo.CableDriver).To(Equal("libreswan"))
-				Expect(brokerInfo.CatalogChannel).To(BeEmpty())
+				Expect(brokerInfo.CatalogChannel).To(Equal(submarinerbrokerinfo.DefaultCatalogChannel))
```

Comment:
The general unit test pattern in the addon is to avoid relying on values provided by the package being tested:

```suggestion
				Expect(brokerInfo.CatalogChannel).To(Equal("stable-0.15"))
```